migrations/20181009110334_added_changelog_table.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('changelog')
+    .createTable('changelog', table => {
+      table.uuid('message_id').primary().notNull();
+      table.uuid('changed_by').references('id').inTable('profiles').notNull();
+      table.integer('establishment_id').references('id').inTable('establishments');
+      table.uuid('model_id').notNull();
+      table.string('action').notNull();
+      table.string('model_type').notNull();
```

Comment:
Really OCD but can we have `model_id` and `model_type` next to each other? :D

Reply:
Just makes it easier when scanning rows in the table via a query tool.

Reply:
👍